src/helm/common/request.py

```diff
@@ -65,16 +70,36 @@ class Request:
     """Multimodal prompt with media objects interleaved (e.g., text, video, image, text, ...)"""
 
     @property
-    def model_organization(self) -> str:
-        """Example: 'openai/davinci' => 'openai'"""
-        model: Model = get_model(self.model)
-        return model.organization
+    def model_host(self) -> str:
+        """Returns the model host (referring to the deployment).
+        Not to be confused with the model creator organization (referring to the model).
+        Example: 'openai/davinci' => 'openai'
+                 'together/bloom' => 'together'"""
+        return self.model_deployment.split("/")[0]
 
     @property
     def model_engine(self) -> str:
-        """Example: 'openai/davinci' => 'davinci'"""
-        model: Model = get_model(self.model)
-        return model.engine
+        """Returns the model engine (referring to the model).
+        This is often the same as self.model_deploymentl.split("/")[1], but not always.
+        For example, one model could be served on several servers (each with a different model_deployment)
+        In that case we would have for example:
+        'aws/bloom-1', 'aws/bloom-2', 'aws/bloom-3' => 'bloom'
+        This is why we need to keep track of the model engine with the model metadata.
+        Example: 'openai/davinci' => 'davinci'"""
+        return self.model.split("/")[1]
+
+    # If only the model_deployment has been provided, then infer the model from the model_deployment
+    # Otherwise check that the deployment correctly implements the model.
+    def __post_init__(self):
+        deployment: ModelDeployment = get_model_deployment(self.model_deployment)
+        if self.model == "":
+            # Bypassing the frozen attribute to set the model
+            object.__setattr__(self, "model", deployment.model_name)
+        elif deployment.model_name != self.model:
+            raise ValueError(
+                f"Model deployment {self.model_deployment} does not implement model {self.model}. "
+                f"Available models: {deployment.model_name}"
+            )
```

Comment:
Shouldn't do this because it's going to crash `helm-summarize` when I'm deserializing existing `Request`s from previous versions because those requests don't contain `model_deployment`.

Also dataclass initialization shouldn't have external dependencies, anyway.